docs/UserGuide.md

```diff
@@ -440,7 +516,21 @@ Format: `reminder add c/CONTACT_INDEX m/MESSAGE d/DATETIME`
 * The datetime must be in the format `yyyy-MM-dd HH:mm`
 
 Examples:
-* `reminder add c/2 m/Send follow-up email d/2020-10-30 15:00` Adds a reminder associated with the 2nd contact that is scheduled for 30th October 2020 3PM, with the message `Send follow-up email`
+* `reminder add c/2 m/Send email to follow up d/2020-10-30 15:00` Adds a reminder associated with the 2nd contact
+ that is scheduled for 30th October 2020 3PM, with the message `Send email to follow up`
+ 
+#### Deleting a reminder: `reminder delete` \[Sebastian Toh Shi Jian\]
+
+Deletes the specified reminder from StonksBook.
+
+Format: `reminder delete INDEX`
+
+* Deletes the reminder at the specified `INDEX`.
+* The index refers to the index number shown in the displayed reminders list.
+* The index must be a positive integer 1, 2, 3, …​
+
+Examples:
+* `reminder list` followed by `reminder delete 2` deletes the 2nd reminder in StonksBook.
```

Comment:
Just wondering if all the examples should all be step-by-step.

Reply:
Yep, I think so. I plan to do them over the weekend, don't think I can finish it by tonight lol